non-critical-infra/flake-module.nix

```diff
@@ -49,8 +50,14 @@
     };
 
   perSystem =
-    { pkgs, inputs', ... }:
+    { inputs', ... }:
+    # Use the latest packages from `nixpkgs-unstable` for dev tools.
```

Comment:
It's entirely non-obvious from this PR: I need sops `--filename-override` flag, which was introduced in [sops 3.9.0], which is not present in the stable nixos release we use here.

[sops 3.9.0]: https://github.com/getsops/sops/blob/main/CHANGELOG.rst#390